src/scripts/site.js

```diff
@@ -1354,19 +1324,18 @@ var AppInfo = {};
     }
 
     function onWebComponentsReady(browser) {
-        var initialDependencies = [];
-
-        if (!window.Promise || browser.web0s) {
-            initialDependencies.push("bower_components/emby-webcomponents/native-promise-only/lib/npo.src");
-        }
-
         initRequireWithBrowser(browser);
 
         if (self.appMode === 'cordova' || self.appMode === 'android' || self.appMode === 'standalone') {
             AppInfo.isNativeApp = true;
         }
 
-        require(initialDependencies, init);
+        if (!window.Promise || browser.web0s) {
```

Comment:
why this was moved down?

Reply:
This was moved down so i could load the dependencies statically when that condition could be met. If that condition fails, it doesn't need any dependencies to be loaded, so a call the callback init directly.